br/pkg/task/restore.go

```diff
@@ -486,7 +486,7 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 	// to avoid import cycle problem which we won't do it in this pr, then refactor this
 	//
 	// if it's point restore and reached here, then cmdName=FullRestoreCmd and len(cfg.FullBackupStorage) > 0
-	if cmdName == FullRestoreCmd && cfg.WithSysTable {
```

Comment:
cannot remove this, if we `InitFullClusterRestore` we will check whether cluster is fresh & whether sys table compatible

Reply:
I don't learn all code, but it doesn't need  `cfg.WithSysTable` here before https://github.com/pingcap/tidb/pull/36652/files#diff-7623fc7b2d82b31b389e3575c938daf9f5724962391d8cb21968b541c45c3191R489

Reply:
I don't understand these code clearly, should  the `rc.fullClusterRestore`  be `!explicitFilter && rc.IsFull() && !cfg.WithSysTable`? Only restoring systbale, br need to check ifcluster is fresh and sys table compatiblable